### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by Annelisebx**

### CODE REVIEW

File: client/src/components/Messaging/Conversation.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    details: {
+        display: 'inline',
+        textAlign: 'center',
+        width: '50vw',
+        backgroundColor: '#c2debd',
+        color: '#3d804f',
+    },
+}));
+
+export default function PopularItem({ event }) {
```
Comment (panzerama) [inline client/src/components/Messaging/Conversation.js:45]: Why is the component for a conversation named PopularItem?

File: client/src/components/Messaging/ConversationContainer.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        marging: '10px',
+        justifyContent: 'space-evenly',
+        flexWrap: 'wrap',
+    },
+    conversation: {
+        margin: theme.spacing(1),
+    },
+}));
+
+export default function PopularItemContainer({events}) {
```
Comment (panzerama) [inline client/src/components/Messaging/ConversationContainer.js:17]: Same question: what's the naming logic here?

File: service/models/Conversation.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+const conversationSchema = new mongoose.Schema({
```
Comment (panzerama) [inline service/models/Conversation.js:3]: This seems like a message schema, but not a conversationn.

File: client/src/components/Messaging/Conversation.js (hunk 1 above)
Comment (Annelisebx) [reply client/src/components/Messaging/Conversation.js:45]: These commits were not supposed to be on this pull request. It was supposed to be for the accessibility additions only. I was copying our code from loading product details to attempt to get messages to store and display. I changed the naming, but it is still incomplete. I will try to change the pull request.

**Comment by Annelisebx** [comment]
Re-submitting in separate PR

**Pull request closed without merging**